master/internal/api_experiment.go

```diff
@@ -10,14 +10,15 @@ import (
 	"strings"
 	"time"
 
+	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 	"google.golang.org/protobuf/encoding/protojson"
 
 	"github.com/pkg/errors"
 
 	"github.com/determined-ai/determined/master/internal/db"
-	"github.com/determined-ai/determined/master/internal/grpc"
+	internalGrpc "github.com/determined-ai/determined/master/internal/grpc"
```

Comment:
i'd be ok with renaming this package `grpcutil` or something

Reply:
Yeah I think that's good, especially if we're gonna import the google.golang.org/grpc into api_trials now too. I see even our grpc package imports the top-level grpc package from upstream, so that conflict is all over.

Reply:
Since I moved the original version of connectionIsClosed to grpcutil, there's no longer the immediate need to rename one of the grpc pacakges, but I went ahead with the rename: there's a bunch of places where we import internal/grpc AND child packages of google.golang.org/grpc. It's confusion waiting to happen...